Fabric.Authorization.API/Dockerfile

```diff
@@ -3,7 +3,11 @@ ARG source
 WORKDIR /app
 EXPOSE 5004
 
-RUN apt-get update -y && \
+RUN echo "deb [check-valid-until=no] http://cdn-fastly.deb.debian.org/debian jessie main" > /etc/apt/sources.list.d/jessie.list
```

Comment:
you should still do the "apt-get update" after these sources are added so that it gets the latest packages.

Reply:
i see the update below